flink-dist/src/test/bin/runBashJavaUtilsCmd.sh

```diff
@@ -36,6 +36,5 @@ FLINK_DIST_JAR=`find $FLINK_TARGET_DIR -name 'flink-dist*.jar'`
 
 . ${bin}/../../main/flink-bin/bin/config.sh > /dev/null
 
-output=$(runBashJavaUtilsCmd ${COMMAND} ${FLINK_CONF_DIR} "$FLINK_TARGET_DIR/bash-java-utils.jar:$FLINK_DIST_JAR}" | tail -n 2)
-extractExecutionParams "$(echo "$output" | head -n 1)"
-extractExecutionParams "$(echo "$output" | tail -n 1)"
+output=$(runBashJavaUtilsCmd ${COMMAND} ${FLINK_CONF_DIR} "$FLINK_TARGET_DIR/bash-java-utils.jar:$FLINK_DIST_JAR}")
+extractExecutionResults "${output}"
```

Comment:
nit: it might make sense to test `extractLoggingOutputs` as well, to not do it manually.